openslides/core/views.py

```diff
@@ -35,6 +36,14 @@ class IndexView(utils_views.View):
     to the custom staticfiles directory. See STATICFILES_DIRS in settings.py.
     """
 
+    @classmethod
```

Comment:
I think we should choose the Django way for decorating (non mixin) classes. That's all.